docs/included.md

```diff
@@ -271,6 +279,14 @@
     [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Binary_Contingency_Scores.html)
   - 
     [False alarm ratio (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#FAR)
+* -  
+    - Forecast rate
```

Comment:
For consistency with the rest of the table, should this be "Forecast Rate" (with a capital "R")?